site/src/pages/components/popover-hint.research.explainer.mdx

```diff
@@ -0,0 +1,169 @@
+These two capabilities are orthogonal but closely related, and this explainer therefore discusses them both.
+
+The motivating use case, but certainly not the only use case, for these two capabilities is in the construction of "tooltips". Typically, tooltips are transient bits of additional information related to a bit of content. This additional information is *not required* and is supplementary in nature. Usually the user accesses this tooltip by hovering, keyboard-focusing, or long-pressing a control.
+
+# "Hover" triggering
+
+The existing popover API includes the concept of "invoking elements":
+
+```html
+<button popovertarget=foo popovertargetaction=show>
+  Click me to show a popover
+</button>
+<div popover id=foo> I'm a popover </div>
+```
+
+The `popovertarget` attribute can be used on buttons (and `<input type=button>`), and it points to the idref of an element with the `popover` attribute. In this configuration, clicking the button automatically shows the popover, without requiring Javascript. In addition, this linkage ensures that the accessibility tree is properly updated to represent the connection. The `popovertargetaction` attribute determines what action to take when the button is activated: `show`, `hide`, or `toggle`.
+
+## Add the `hover` value
+
+To support hover triggering, a new value is added: `popovertargetaction=hover`. When set to the `hover` value, the popover will be **shown** when the invoking element is **a)** hovered with the mouse, **b)** focused, or **c)** long-pressed (mobile).
+
+## CSS values to control delay
+
+The majority of hover-triggered UI has a set of delays associated with it. It tends to be jarring to the user if UI elements show up *immediately* upon hovering an element. For example, that causes elements to show and hide rapidly as the user moves their mouse across the screen. To allow the user to be more intentional, there is an expectation that the element of interest must be hovered for a short period of time before any popovers are triggered. Additionally, it is common for such hover-triggered popovers to also be hidden automatically when they are no longer hovered, again after a sufficient delay. This "hide delay" is particularly important in the case that there is a physical gap between the invoking element and the hint popover: the user must move the mouse from the invoker to the popover and cross that gap, and it is frustrating if the popover closes while that traversal is taking place.
+
+To facilitate these delays, two new CSS properties are added:
+
+- `popover-show-delay` - applied to the invoking element. The invoking element must be hovered, focused, or long-pressed for this long before the popover is shown. The default value for this property is `0.5s`.
+- `popover-hide-delay` - applied to the popover. The invoking element *or* the popover must be **not** hovered, focused, or long-pressed for this long before the popover is hidden. The default value for this property is `infinity`.
```

Comment:
Interesting feedback! I had envisioned these to work with popovers (and maybe dialogs?) only. How do you envision making them more general?

As to `popover-show-delay` I'm not sure this can be done with `animation-delay` - can you prove me wrong? My concern is that the behavior wouldn't be de-bounced in that case. I.e. set `popover-show-delay` to 0.5 seconds, and have the user hover and then de-hover the element only for 0.2 seconds. Nothing should happen, but I think `animation-delay` would cause the popover to just be shown after 0.5 seconds, right?

Reply:
https://codepen.io/keithamus/pen/yLQabJK this uses `animation-delay` to ensure that as you hover and de-hover (or focus/defocus) the `button` for less than `0.5s`, the `div` will not be shown. I believe this is properly debounced as you describe. I don't know how to get the same effect upon exit of that state; transitions don't work quite the same, I believe.

Reply:
Ahh nice - I see that you get that by adding `:focus` to clear the animation if you de-hover. But you're right - not possible on the de-hover side of things.

Do you think this is worth opening a CSSWG issue to discuss? I mean about the "more general problem" question?